web_external/Datasets/ApplyMetadataView.js

```diff
@@ -143,6 +147,9 @@ isic.views.ApplyMetadataView = isic.View.extend({
         }
     },
 
+    /**
+     * @param {isic.models.DatasetModel} settings.dataset
+     */
```

Comment:
If we're feeling ambitious we could consider using JSDoc format consistently throughout the client code.